compiler-rt/lib/scudo/standalone/CMakeLists.txt

```diff
@@ -108,6 +108,7 @@ set(SCUDO_SOURCES
   flags.cpp
   fuchsia.cpp
   linux.cpp
+  linux_common.cpp
```

Comment:
Given that we have `report.h` to report platform independent errors, I'm thinking that we may want the naming to be something like `report_linux_common.h` (and linux.cpp will be deprecated)

Reply:
I decided to change this to report_linux since having common in the name seems redundant.